src/meta-srv/src/procedure/region_migration.rs

```diff
@@ -788,8 +788,8 @@ mod tests {
         }
     }
 
-    #[test]
-    fn test_data_serialization() {
+    #[tokio::test]
+    async fn test_data_serialization() {
```

Comment:
Why these two tests need to be async? There are no async functions calls inside.